### ISSUE TITLE
Setting empty snapshot directory with Rely causes a confusing error

### ISSUE THREAD

**Issue by michaelboulton** [facebookexperimental/reason-native#166]
For example:
```reason
projectDir = ...

include Rely.Make({
  let config =
    Rely.TestFrameworkConfig.initialize({
      snapshotDir: "",
      projectDir,
    });
});

```
Causes
```
Fatal error: exception Unix.Unix_error(Unix.ENOENT, "mkdir", "")
```
I just upgraded from 1.3.1 to 2.1.0 and it wasn't happening with the old version. I'm not creating any snapshots which is why I set it to an empty string.

**Comment by bandersongit** [comment]
Thanks for filing! I remember making a change where if the snapshot directory didn't exist we created it because of issues with writing snapshots in that case. I'll improve the error message here. You aren't the first person to mention they aren't using snapshots, so I'll definitely considering making them optional come the next major version

**Comment by kyldvs** [comment]
I think making it optional and failing at runtime would be good for a future version

**Comment by bandersongit** [comment]
An improved failure message for this will be included in the 2.2.0 release which will be cut today, and definitely considering making this optional in the future
